src/remixDClient.js

```diff
@@ -0,0 +1,33 @@
+const services = require('./services')
+const path = require('path');
+
+class RemixDClient {
+  // List of methods exposed by the client (required for error checking)
+  constructor (services) {
+    this.services = services || []
+    this.sharedFolder = path.resolve(__dirname, '..', process.env.SHARED_FOLDER)
+  }
+
+  register (service) {
+    this.services.push(service)
+  }
+
+  call (message, callback) {
+    const {action, id, key, name, payload} = message
+    try {
+      const svc = this.services[name]
+
+      if (name === 'fileManager') {
+        svc[key](payload, callback)
+      } else {
+        this.services["fileManager"]["createFolder"](this.sharedFolder) //Create folder if it doesn't exist
```

Comment:
could we do that one in another PR if really needed?

Reply:
This has been removed and will be added to another PR.